test/player_integration.js

```diff
@@ -87,6 +88,31 @@ describe('Player', () => {
     });
   });  // describe('attach')
 
+  describe('set mediaElement.currentTime in manifestparsed', () => {
+    it('does not get segments prior to mediaElement.currentTime', async () => {
+      // if (shaka.util.Platform.isApple()) {
+      //   pending('setting currentTime during load is not supported by apple');
+      // }
```

Comment:
If this test fails on Safari, we may need to skip it or fix it before merging.  We run tests on Safari nightly in our lab.